src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala

```diff
@@ -229,7 +229,7 @@ private[redshift] object Parameters {
     /**
      * Extra options to append to the Redshift COPY command (e.g. "MAXERROR 100").
      */
-    def extraCopyOptions: String = parameters.get("extracopyoptions").getOrElse("")
+    def extraCopyOptions: String = parameters.getOrElse("extracopyoptions", "")
```

Comment:
Why did you change this? I think it was intended to be `get ('extracopyoptions')` and if it's not defined, just return an empty String

Reply:
It just more elegant way of writing. More cleaner I think.
The IDE was also giving hints to write it that way.